### ISSUE TITLE
"Git: Accept Merge" not accessible

### ISSUE THREAD

**Issue by cgebbe** [microsoft/vscode-remote-release#7201]
We have written the needed data into your clipboar
Type: <b>Bug</b>

- GIVEN a merge conflict
- WHEN opening conflicting files via the sources control (in the mixed layout)
- THEN the EXPECTED behavior is that an "Accept Merge" button appears and that the quick command "Git: Accept Merge" is available (this works with a local repository).
- Instead, the ACTUAL behavior is that neither the button appears nor the quick command exists (this happens with a remote repository).

I tried some workarounds but couldn't find any. Therefore, the nice 3-way overview (incoming, current, result) seems to be unusable using the ssh-remote extension, which is a real pity :(.

Extension version: 0.84.0
VS Code version: Code 1.71.0 (784b0177c56c607789f9638da7b6bf3230d47a8c, 2022-09-01T07:36:10.600Z)
OS version: Windows_NT x64 10.0.19042
Modes:
Sandboxed: No
Remote OS version: Linux x64 5.4.0-122-generic
Remote OS version: Linux x64 5.4.0-122-generic

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-10610U CPU @ 1.80GHz (8 x 2304)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled<br>webgpu: disabled_off|
|Load (avg)|undefined|
|Memory (System)|31.73GB (15.85GB free)|
|Process Argv|C:/git/_me/toolbox/journal/20220916_Friday.md C:/git/_me/toolbox --crash-reporter-id 3e962266-f2a7-4163-99a6-459d0190a0c8|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|SSH: gpuserver6.definiens.local|
|OS|Linux x64 5.4.0-122-generic|
|CPUs|AMD EPYC 7F72 24-Core Processor (96 x 1799)|
|Memory (System)|503.81GB (475.19GB free)|
|VM|0%|

|Item|Value|
|---|---|
|Remote|SSH: gpuserver6.definiens.local|
|OS|Linux x64 5.4.0-122-generic|
|CPUs|AMD EPYC 7F72 24-Core Processor (96 x 1799)|
|Memory (System)|503.81GB (475.19GB free)|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383:30185418
vspor879:30202332
vspor708:30202333
vspor363:30204092
vslsvsres303:30308271
pythonvspyl392:30443607
vserr242cf:30382550
pythontb:30283811
vsjup518:30340749
pythonptprofiler:30281270
vshan820:30294714
vstes263:30335439
vscoreces:30445986
pythondataviewer:30285071
vscod805cf:30301675
binariesv615:30325510
bridge0708:30335490
bridge0723:30353136
cmake_vspar411cf:30557515
vsaa593cf:30376535
pythonvs932:30410667
cppdebug:30492333
vsclangdf:30486550
c4g48928:30535728
hb751961:30553087
dsvsc012:30540252
azure-dev_surveyone:30548225
2144e591:30553903
fc301958:30567733

```

</details>

d because it was too large to send. Please paste.

**Comment by jrieken** [comment]
> this happens with a remote repository

/duplicate

**Comment by jrieken** [comment]
and fixed in 1.72.2
